src/command/cheque/deposit.ts

```diff
@@ -0,0 +1,41 @@
+import { Argument, LeafCommand } from 'furious-commander'
+import { bold, green } from 'kleur'
+import { ChequeCommand } from './cheque-command'
+
+export class Deposit extends ChequeCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'deposit'
+
+  public readonly aliases = ['dep']
+
+  public readonly description = 'Deposit tokens from overlay address into chequebook'
+
+  @Argument({
+    key: 'amount',
+    type: 'string',
+    describe: 'Amount of tokens to deposit (must be positive integer)',
+    required: true,
+  })
+  public amount!: string
+
+  public async run(): Promise<void> {
+    super.init()
+
+    if (!(await this.checkDebugApiHealth())) {
+      return
+    }
+
+    const amount = this.parsePositiveBigInt(this.amount)
+
+    if (!amount) {
+      this.console.error('Could not parse amount. Is it a positive number?')
```

Comment:
A better message could be:
`Invalid amount. The amount must be a positive integer.`